Library/OcCpuLib/OcCpuLib.c

```diff
@@ -1177,14 +1180,16 @@ ScanAmdProcessor (
   //
   Cpu->AppleProcessorType = AppleProcessorTypeCorei5Type5;
   //
-  // get TSC Frequency calculated in OcTimerLib
+  // get TSC Frequency calculated in OcTimerLib, only if we get it already when it under visualization.
```

Comment:
`Get TSC Frequency calculated in OcTimerLib, unless we got it already from virtualization extensions.`